DSL/Search.php

```diff
@@ -69,7 +69,7 @@ class Search
     /**
      * @var string|null
      */
-    private $scrollDuration;
+    private $scroll;
```

Comment:
`scroll` is completely fine. Even in elasticsearch itself this parameter is named `scroll` not `scrollDuration`